<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VulnerabilityAssessment</class>
 <widget class="QMainWindow" name="VulnerabilityAssessment">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>850</width>
    <height>778</height>
   </rect>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::DefaultContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>VulnerabilityAssessment</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>841</width>
      <height>801</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="select_label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>321</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Select one of the IP Camera below:</string>
     </property>
    </widget>
    <widget class="QListWidget" name="detected_ip_camera_listWidget">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>50</y>
       <width>771</width>
       <height>111</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QLabel" name="provide_label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>180</y>
       <width>491</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Provide Addtional IP Camera Information if Applicable:</string>
     </property>
    </widget>
    <widget class="QLabel" name="vendor_label">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>220</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Vendor:</string>
     </property>
    </widget>
    <widget class="QLabel" name="firmware_label">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>250</y>
       <width>171</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Firmware Version:</string>
     </property>
    </widget>
    <widget class="QLabel" name="nvd_label">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>290</y>
       <width>141</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>NVD API Key:</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="vendor_textEdit">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>210</y>
       <width>511</width>
       <height>31</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>Vendor</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="firmware_textEdit">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>250</y>
       <width>511</width>
       <height>31</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>Firmware Version</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="nvd_textEdit">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>290</y>
       <width>511</width>
       <height>31</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>NVD API Key</string>
     </property>
    </widget>
    <widget class="QLabel" name="not_applicable_label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>330</y>
       <width>551</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>* If not applicable, just leave it empty and proceed</string>
     </property>
    </widget>
    <widget class="QPushButton" name="vs_pushButton">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>370</y>
       <width>221</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Vulnerability Scanning</string>
     </property>
    </widget>
    <widget class="QPushButton" name="bfa_pushButton">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>370</y>
       <width>221</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Brute Force Attack</string>
     </property>
    </widget>
    <widget class="QLabel" name="note_label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>430</y>
       <width>761</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>*Note: One time can only process with one IP Camera, if you have multiple cameras, select again on</string>
     </property>
    </widget>
    <widget class="QLabel" name="note_label_2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>450</y>
       <width>761</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>the list above and click the button again!</string>
     </property>
    </widget>
    <widget class="QLabel" name="detected_vulnerabilities_label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>510</y>
       <width>491</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Detected Vulnerabilities List:</string>
     </property>
    </widget>
    <widget class="Line" name="line">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>490</y>
       <width>841</width>
       <height>16</height>
      </rect>
     </property>
     <property name="lineWidth">
      <number>2</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QListWidget" name="detected_vulnerabilities_listWidget">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>550</y>
       <width>781</width>
       <height>111</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QPushButton" name="mitigation_pushButton">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>680</y>
       <width>221</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Mitigation</string>
     </property>
    </widget>
    <widget class="QPushButton" name="va_back_pushButton">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>680</y>
       <width>221</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Back</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>850</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
